e2e_tests/passing/happy_path/user_fills_resetting_slot.yml

```diff
@@ -20,11 +20,3 @@ test_cases:
       - utter: utter_can_do_something_else
       - user: Bye then
       - utter: utter_goodbye
-      - user: I would like to confirm my account
```

Comment:
Isn't this part here for checking that the slot values are reset and that they need to be filled again?

Reply:
can we save the steps here and just whether the slots in question were set to None?